next.js

```diff
@@ -6,4 +6,40 @@ module.exports = {
 		'./react.js',
 		'plugin:@next/next/core-web-vitals',
 	],
+	plugins: ['simple-import-sort', 'unused-imports'],
+	rules: {
+		'import/no-extraneous-dependencies': 'off',
+		'import/order': 'off',
+		'simple-import-sort/imports': [
+			'warn',
+			{
+				groups: [
+					['^((?!((.*).(scss|css))|assets|src|./|../|@[^storybook|^sentry]))'],
+					['^(@[^storybook|^sentry](.*))'],
+					['^(@apps/|apps/|@libs/)'],
+					['^(components/)'],
+					['^(config/)'],
+					['^(lib/)'],
+					['^(pages/)'],
+					['^(state/)'],
+					['^(styles/)'],
+					['^(translations/)'],
+					['^(types/)'],
+					['^(utils/)'],
+					['^(assets/|(.*).(scss|css)$)'],
+					['^((./|../))'],
+				],
+			},
+		],
```

Comment:
I think this is a better fit for an overwrite in specific projects - maybe good to add to the boilerplate.